website/docs/integrator-section/webapp-tutorial/revoke-document-store-webapp.md

```diff
@@ -1,31 +1,43 @@
 ---
 id: revoke-document-store-webapp
-title: Revoke Document with Web app
-sidebar_label: Revoke Document Store
+title: Revoking Document with Webapp
```

Comment:
Web App should be two words. Maybe something like below? Not sure how the other sections are to be titled.

```suggestion
title: Revoking a Document with a Web App
```

Reply:
Ran a global search for "webapp" and updated all to "web app"